src/forms/BoundForm.tsx

```diff
@@ -76,7 +76,7 @@ export function BoundForm<F>(props: BoundFormProps<F>) {
 
   return (
     <div {...tid}>
-      <FormLines labelSuffix={{ required: "*" }} width="full" gap={4}>
+      <FormLines labelSuffix={{ required: "*" }} width="full" gap={3.5}>
```

Comment:
Given this `labelSuffix` is now the default, does it make sense to remove here?

Reply:
I'd think so 🤔